### ISSUE TITLE
On the main menu of UNA, screen reader doesn't indicate buttons/images that can be selected.

### ISSUE THREAD

**Issue by prajwalmanju** [microsoft/AdaptiveCards#6777]
### Target Application

Microsoft Teams

### Application Operating System

Windows

### Schema Version

v1.2

### Problem Description

On the main menu of UNA, screen reader reads that out the menu items but it doesn't indicate they are buttons/images that can be selected. In addition, after they are read, user cannot navigate or select any of the button/image afterwards.

### Expected Outcome

screen reader should indicate they are buttons/images that can be selected. In addition, after they are read, user should navigate or select any of the button/image afterwards.

### Actual Outcome

screen reader doesn't indicate they are buttons/images that can be selected. In addition, after they are read, user cannot navigate or select any of the button/image afterwards.

### Card JSON

```JSON
{
  "type": "AdaptiveCard",
  "body": [
    {
      "type": "ColumnSet",
      "columns": [
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": "Navigation and Help",
              "selectAction": {
                "type": "Action.Submit",
                "title": "Navigation and help",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with Navigation and Help"
                  }
                }
              },
              "horizontalAlignment": "Center",
              "width": "100px"
            }
          ],
          "height": "stretch",
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        },
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": " Pay and benefits",
              "horizontalAlignment": "Center",
              "selectAction": {
                "type": "Action.Submit",
                "title": "Pay and Benefits",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with Pay and benefits"
                  }
                }
              },
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        },
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": " Learning and development",
              "selectAction": {
                "type": "Action.Submit",
                "title": "Learning and development",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with Learning and development"
                  }
                }
              },
              "horizontalAlignment": "Center",
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        }
      ],
      "style": "default",
      "horizontalAlignment": "Center",
      "spacing": "Padding",
      "height": "stretch"
    },
    {
      "type": "ColumnSet",
      "columns": [
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": " joining moving and leaving",
              "selectAction": {
                "type": "Action.Submit",
                "title": "Joining Moving and leaving",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with joining moving and leaving"
                  }
                }
              },
              "horizontalAlignment": "Center",
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        },
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": "tools",
              "selectAction": {
                "type": "Action.Submit",
                "title": "Tools",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with the tools you have"
                  }
                }
              },
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        },
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": "my workplace",
              "horizontalAlignment": "Center",
              "selectAction": {
                "type": "Action.Submit",
                "title": "My Workplace",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with my workplace"
                  }
                }
              },
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        }
      ],
      "style": "default",
      "horizontalAlignment": "Center",
      "spacing": "Padding",
      "height": "stretch"
    },
    {
      "type": "ColumnSet",
      "columns": [
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": "finance",
              "selectAction": {
                "type": "Action.Submit",
                "title": "Finance",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with Finance"
                  }
                }
              },
              "horizontalAlignment": "Center",
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        },
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": "My Una",
              "selectAction": {
                "type": "Action.Submit",
                "title": "My Una",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with My Una?"
                  }
                }
              },
              "horizontalAlignment": "Center",
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        },
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": "My Travel",
              "selectAction": {
                "type": "Action.Submit",
                "title": "Learning and development",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with Travel"
                  }
                }
              },
              "horizontalAlignment": "Center",
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        }
      ],
      "style": "default",
      "horizontalAlignment": "Center",
      "spacing": "Padding",
      "height": "stretch"
    }
  ],
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2"
}
```

**Comment by Wajeed-msft** [comment]
This issue has been copied over to Microsoft Teams repo here: https://github.com/MicrosoftDocs/msteams-docs/issues/4732
